src/core/dfn.js

```diff
@@ -237,4 +237,37 @@ function addContractDefaults() {
   for (const dfn of exportableDfns) {
     dfn.dataset.export = "";
   }
+
+  // - Sets data-defines on well-known definition content patterns
```

Comment:
let's move this to it's own function... this function is already getting a bit too long.